experimenter/experimenter/nimbus_ui_new/templatetags/nimbus_extras.py

```diff
@@ -65,3 +67,21 @@ def format_not_set(value):
     if value in [None, "", "NOT SET"]:
         return mark_safe('<span class="text-danger">Not set</span>')
     return value
+
+
+@register.filter(name="format_json")
+def format_json(value):
+    """Formats the JSON value for display by pretty-printing the JSON,
+    and if the JSON is empty or None, it calls format_not_set."""
+    if value in [None, "", "NOT SET"]:
```

Comment:
If we're gonna do this check twice

```py
if value in [None, "", "NOT SET"]
```

Then we should move that to a constant the top of the file like

```py
NOT_SET_VALUES = [None, "", "NOT SET"]

...

if value not in NOT_SET_VALUES:
```